src/rtdev.erl

```diff
@@ -765,6 +778,26 @@ get_node_logs() ->
           {lists:nthtail(RootLen, Filename), Port}
       end || Filename <- filelib:wildcard(Root ++ "/*/dev/dev*/log/*") ].
 
+get_node_debug_logs() ->
+    NodeMap = rt_config:get(rt_nodes),
+    rt:pmap(fun(Node) ->
+                    get_node_debug_logs(Node)
+            end,
+            NodeMap).
+
+get_node_debug_logs({_Node, NodeNum}) ->
+    Path = relpath(node_version(NodeNum)),
+    Args = ["--logs"],
+    Cmd = riak_debug_cmd(Path, NodeNum, Args),
+    file:delete(?DEBUG_LOG_FILE(NodeNum)),
```

Comment:
Just by inspection, this code feels slightly odd. I'm guessing I'm just missing something, but it looks like we delete the log file before we actually spawn the command to upload it. Even if everything works fine here, I'd like to see a comment or two explaining more clearly what's actually going on.